nettest.js: avoid confusing desktop users

agrabeli opened the pull request:
This OONI Run landing page appears if you access an OONI Run link on desktop or if you don't have the mobile app already installed: https://run.ooni.io/nettest?tn=web_connectivity&ta=%7B%22urls%22%3A%5B%22http%3A%2F%2F%22%5D%7D&mv=1.2.0 

I therefore propose that we delete everything in this page & only include a simple instruction telling the user to install the OONI Probe mobile app in order to proceed with using an OONI Run link.

Closes https://github.com/ooni/ooni.org/issues/551

hellais commented on pages/nettest.js:
```diff
@@ -151,18 +151,10 @@ export default class extends React.Component {
-          <Text pt={2} pb={3}>If you have the OONI Probe app installed, click Run to start the test.</Text>
-
-          <Link href={deepLink}>
-            <Button>Run</Button>
```
The deep link must remain, because in the event that it does not trigger mobile deep linking because we failed to detect that they are on mobile (or the plethora of ways in which it might fail), you need to be able to override it with a clickable button which has the `ooni://` prefix.

agrabeli replied:
Oh, thanks for the heads-up! I've kept this text, and made some edits.

hellais commented on pages/nettest.js:
```diff
@@ -151,18 +151,10 @@ export default class extends React.Component {
-          <Text pt={2}>Measure internet censorship by running OONI Probe.</Text>
 
-          <Heading pt={2} h={2}>You have the app</Heading>
-          <Text pt={2} pb={3}>If you have the OONI Probe app installed, click Run to start the test.</Text>
```
Again I that all this text should be left because of what I mentioned below.

agrabeli replied:
OK, thanks.

hellais commented on pages/nettest.js:
```diff
@@ -151,18 +151,10 @@ export default class extends React.Component {
+          <Heading pt={4} h={2}>Install the OONI Probe mobile app</Heading>
           <Text pt={2} pb={3}>
-            If you are on a desktop computer, click Install to learn how you can install OONI Probe.
+            Currently, OONI Run links only work with the OONI Probe mobile app. 
```
This seems like a good copy edit to make. I suggest doing a commit with just this small copy edit.

agrabeli replied:
OK, thanks!

agrabeli commented:
@hellais Thanks for the review! I've addressed your feedback here: https://github.com/ooni/run/pull/45/commits/29ca06d28566f7d5772df498f511544294bd9957

sarathms commented:
![image](https://user-images.githubusercontent.com/700829/92165273-cc1db400-ee04-11ea-9ce1-ecace4b00e35.png)

mobile:
<img src='https://user-images.githubusercontent.com/700829/92166369-819d3700-ee06-11ea-9d34-410ac3bb3564.png' height=300 />

sarathms reviewed (approved):
:whale: This looks good to me. Some indentation was out of place. My editor fixed it, so I pushed it in here itself.
